www/index.html

```diff
@@ -189,37 +189,57 @@
+                                This application needs a ZIM archive to work. If you did not download one yet, please see the About section
+                                <div id="openLocalFiles" style="display: none;">
+                                    Please select the .zim file (or all the .zimaa, .zimab etc in case of a split ZIM file)<br />
+                                    <input type="file" id="archiveFiles" multiple class="btn" accept=".zim,.dat,.idx,.txt,.zimaa,.zimab,.zimac,.zimad,.zimae,.zimaf,.zimag,.zimah,.zimai,.zimaj,.zimak,.zimal,.zimam,.ziman,.zimao,.zimap,.zimaq,.zimar,.zimas,.zimat,.zimau,.zimav,.zimaw,.zimax,.zimay,.zimaz,.zimba,.zimbb,.zimbc,.zimbd,.zimbe,.zimbf,.zimbg,.zimbh,.zimbi,.zimbj,.zimbk,.zimbl,.zimbm,.zimbn,.zimbo,.zimbp,.zimbq,.zimbr,.zimbs,.zimbt,.zimbu,.zimbv,.zimbw,.zimbx,.zimby,.zimbz" /><br />
+                                    <strong>Only Wikimedia contents (wiki*.zim* files) have been tested</strong> for now. There are known bugs on other contents, and audio/video/dynamic contents are not supported for now.<br />
+                                </div>
+                                <div id="scanningForArchives" style="display: none;">
+                                    <br /> Scanning for archives... Please wait <img src="img/spinner.gif" alt="Please wait..." />
+                                </div>
+                                <div id="chooseArchiveFromLocalStorage" style="display: none;">
+                                    <br /> Please select the archive you want to use : <select id="archiveList" class="form-control"></select>
+                                    <br /> Click <a id="btnRescanDeviceStorage">here</a> to rescan your SD Cards and internal memory
+                                </div>
+                            </div>
+                        </div>
                     </div>
-                    <div id="scanningForArchives" style="display: none;">
-                        <br /> Scanning for archives... Please wait <img src="img/spinner.gif" alt="Please wait..." />
-                    </div>
-                    <div id="chooseArchiveFromLocalStorage" style="display: none;">
-                        <br /> Please select the archive you want to use : <select id="archiveList" class="form-control"></select>
-                        <br /> Click <a id="btnRescanDeviceStorage">here</a> to rescan your SD Cards and internal memory
+
+                    <div class="container">
+                        <h3>Expert settings</h3>
+                        <div class="row">
+                            <div class="column">
+                                <div class="panel panel-danger" id="contentInjectionModeDiv">
+                                    <div class="panel-heading">Do not touch unless you know what you're doing!</div>
+                                    <div class="panel-body">
+                                        <p>Content injection mode:</p>
```

Comment:
I think it would be better to put "Content injection mode (do not touch unless you know what you're doing)" as the panel heading, and remove this <p> tag

Reply:
Will do.